src/Symfony/Component/Validator/Constraints/PasswordStrengthEstimatorInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+interface PasswordStrengthEstimatorInterface
```

Comment:
I'm in favor of removing this interface. One less symbol to maintain.

Reply:
@nicolas-grekas OK I removed the interface
@GromNaN The `final` statement has been removed

Reply:
there is no reason to remove the final keyword (if the class is kept), as the suggestion is to wire a closure in the validator.